### PR TITLE
Create Homepage 

### PR DESCRIPTION
**Pull request by ritik307**
### Description:
Made UI template for the homepage

### Task List:
- [x] Added header
- [x] Added dummy end points
- [x] Added footer
- [x] Change the title of the webpage (currently `Document`)
- [x] Resolve conflicts

### Note:
This is not the final homepage. It is the layout of what the page would look like. If you have any changes request do let me know or I will provide it according to my understanding.

### Video:

https://user-images.githubusercontent.com/22374829/112882244-55227280-90ea-11eb-846d-dbc7f6bb7c99.mp4

### CODE REVIEW

**Comment by ritik307** [comment]
@paramsiddharth plz...have a look

**Comment by paramsiddharth** [comment]
> @paramsiddharth plz...have a look

Looks great to me! Please continue the work on the homepage.

Note that since this involves changes to the `package.json`, `package-lock.json`, and `app.ts` files and many pending PRs which will soon be merged do too, before the final merge, you might need to resolve several merge conflicts, so be prepared for that too.

**Comment by ritik307** [comment]
@paramsiddharth sure😄

**Comment by ritik307** [comment]
@paramsiddharth I have made all the changes so plz consider reviewing the PR.

**Comment by paramsiddharth** [comment]
> @paramsiddharth thanks will appreciate that 😁

Impressive.
image
Good job on the UI and the homepage! You've covered pretty much everything.

**Comment by paramsiddharth** [comment]
@ritik307 Only 2 steps to go.

1. Change the title of the webpage. It currently says "Document". It would be nice if you add an icon too, though that won't be necessary.
2. As I had warned, there were bound to be conflicts. There's one in `tasks.ts` after the merge of #65. Please resolve them.

**Comment by ritik307** [comment]
> @ritik307 Only 2 steps to go.
> 
> 1. Change the title of the webpage. It currently says "Document". It would be nice if you add an icon too, though that won't be necessary.
> 2. As I had warned, there were bound to be conflicts. There's one in `tasks.ts` after the merge of #65. Please resolve them.

Got it will do that ASAP

**Comment by ritik307** [comment]
@paramsiddharth Done!

File: src/api/routes/tasks.ts [hunk 1]
```diff
@@ -26,6 +26,16 @@ router.get("/", (req: Request, res: Response, next: NextFunction) => {
...
+//GET request /tasks/index
+router.get("/:index", (req: Request, res: Response, next: NextFunction) => {
+  try {
+    const idx = req.params.index;
+    console.log("index: ", idx)
+    console.log("data: ", data.data.length)
+    
+    res.status(200).json(data.data[idx]);
+
+
```
Comment (paramsiddharth) [inline src/api/routes/tasks.ts:38]: This block is erroneous and seems like an accidental change! Please fix.

**Review by paramsiddharth — CHANGES_REQUESTED** [review]

**Comment by paramsiddharth** [comment]
@ritik307 Almost there.

File: src/api/routes/tasks.ts (hunk 1 above)
Comment (ritik307) [reply src/api/routes/tasks.ts]: have resolved the conflict. plz check

**Comment by ritik307** [comment]
@paramsiddharth have resolved the conflict. plz check

**Comment by paramsiddharth** [comment]
Good job, @ritik307 ! Thank you for contributing!

**Pull request merged**